src/main.c

```diff
@@ -350,10 +388,15 @@ int main(int argc, char **argv)
 			// TODO: Make modules.conf name configurable
 			modules_conf_path = strdup(file->fts_path);
 		} else {
-			print_if_verbose(
-				"Skipping %s which is not a policy file\n",
-				file->fts_path);
-			if (!recursive_scan) {
+			if (recursive_scan) {
```

Comment:
The idea is to make the message more highlighted and unconditional in non-recursive mode?  That makes sense to me, but it too me a while to figure out what the intention was here.  Can you comment this a little more thoroughly to explain the rationale behind the recursive/non-recursive difference in output?

I do feel like we should avoid printing the message twice for directories in both modes though.

Reply:
Reformatted this logic; I hope it is more readable now